packages/okta-react/test/jest/auth.test.js

```diff
@@ -345,4 +382,160 @@ describe('Auth component', () => {
     const authenticated = await auth.isAuthenticated();
     expect(authenticated).toBeFalsy();
   });
+
+  describe('login()', () => {
```

Comment:
I need to save a reference to this test the next time I want a good example of tests in this lib